<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QMdiArea" name="mdiArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>字体</string>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFontComboBox" name="fontComboBox"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>字号</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="sizecomboBox"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>909</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <property name="separatorsCollapsible">
     <bool>false</bool>
    </property>
    <addaction name="actionnew"/>
    <addaction name="actionopen"/>
    <addaction name="separator"/>
    <addaction name="actionsave"/>
    <addaction name="actionsaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionprint"/>
    <addaction name="actionprintPreview"/>
    <addaction name="actionexit"/>
   </widget>
   <widget class="QMenu" name="menubianji">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="actionundo"/>
    <addaction name="actionredo"/>
    <addaction name="separator"/>
    <addaction name="actioncut"/>
    <addaction name="actioncopy"/>
    <addaction name="actionpaste"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>格式</string>
    </property>
    <widget class="QMenu" name="menu_5">
     <property name="title">
      <string>字体</string>
     </property>
     <addaction name="actionbold"/>
     <addaction name="actionitalic"/>
     <addaction name="actionunderline"/>
    </widget>
    <widget class="QMenu" name="menu_6">
     <property name="title">
      <string>段落</string>
     </property>
     <addaction name="actionleft"/>
     <addaction name="actionright"/>
     <addaction name="actioncenter"/>
     <addaction name="actionjustify"/>
    </widget>
    <addaction name="menu_5"/>
    <addaction name="menu_6"/>
    <addaction name="actioncolor"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>窗口</string>
    </property>
    <addaction name="actionclose"/>
    <addaction name="actioncloseAll"/>
    <addaction name="separator"/>
    <addaction name="actiontile"/>
    <addaction name="actioncascade"/>
    <addaction name="separator"/>
    <addaction name="actionnext"/>
    <addaction name="actionprevious"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionabout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menubianji"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>1</width>
     <height>0</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionnew"/>
   <addaction name="actionopen"/>
   <addaction name="actionsave"/>
   <addaction name="actionprintPreview"/>
   <addaction name="separator"/>
   <addaction name="actionundo"/>
   <addaction name="actionredo"/>
   <addaction name="actioncut"/>
   <addaction name="actioncopy"/>
   <addaction name="actionpaste"/>
   <addaction name="separator"/>
   <addaction name="actionbold"/>
   <addaction name="actionitalic"/>
   <addaction name="actionunderline"/>
   <addaction name="actioncolor"/>
   <addaction name="separator"/>
   <addaction name="actionleft"/>
   <addaction name="actionright"/>
   <addaction name="actioncenter"/>
   <addaction name="actionjustify"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
  </widget>
  <action name="actionnew">
   <property name="icon">
    <iconset>
     <normaloff>images/QT.png</normaloff>images/QT.png</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionopen">
   <property name="icon">
    <iconset>
     <normaloff>images/dakai.png</normaloff>images/dakai.png</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionsave">
   <property name="icon">
    <iconset>
     <normaloff>images/baocun.png</normaloff>images/baocun.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionsaveAs">
   <property name="icon">
    <iconset>
     <normaloff>images/lingcun.png</normaloff>images/lingcun.png</iconset>
   </property>
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
  <action name="actionprint">
   <property name="icon">
    <iconset>
     <normaloff>images/dayin.png</normaloff>images/dayin.png</iconset>
   </property>
   <property name="text">
    <string>打印</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionprintPreview">
   <property name="icon">
    <iconset>
     <normaloff>images/yulan.png</normaloff>images/yulan.png</iconset>
   </property>
   <property name="text">
    <string>打印预览</string>
   </property>
  </action>
  <action name="actionexit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionundo">
   <property name="text">
    <string>撤销</string>
   </property>
  </action>
  <action name="actionredo">
   <property name="text">
    <string>恢复</string>
   </property>
  </action>
  <action name="actioncut">
   <property name="text">
    <string>剪切</string>
   </property>
  </action>
  <action name="actioncopy">
   <property name="text">
    <string>复制</string>
   </property>
  </action>
  <action name="actionpaste">
   <property name="text">
    <string>粘贴</string>
   </property>
  </action>
  <action name="actioncolor">
   <property name="text">
    <string>颜色</string>
   </property>
  </action>
  <action name="actionbold">
   <property name="text">
    <string>加粗</string>
   </property>
  </action>
  <action name="actionitalic">
   <property name="text">
    <string>倾斜</string>
   </property>
  </action>
  <action name="actionunderline">
   <property name="text">
    <string>下划线</string>
   </property>
  </action>
  <action name="actionleft">
   <property name="text">
    <string>左对齐</string>
   </property>
  </action>
  <action name="actionright">
   <property name="text">
    <string>右对齐</string>
   </property>
  </action>
  <action name="actioncenter">
   <property name="text">
    <string>居中</string>
   </property>
  </action>
  <action name="actionjustify">
   <property name="text">
    <string>两端对齐</string>
   </property>
  </action>
  <action name="actionclose">
   <property name="text">
    <string>关闭</string>
   </property>
  </action>
  <action name="actioncloseAll">
   <property name="text">
    <string>关闭所有</string>
   </property>
  </action>
  <action name="actiontile">
   <property name="text">
    <string>平铺</string>
   </property>
  </action>
  <action name="actioncascade">
   <property name="text">
    <string>层叠</string>
   </property>
  </action>
  <action name="actionnext">
   <property name="text">
    <string>下一个</string>
   </property>
  </action>
  <action name="actionprevious">
   <property name="text">
    <string>前一个</string>
   </property>
  </action>
  <action name="actionabout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
